### ISSUE TITLE
Build failed with Rust nightly

### ISSUE THREAD

**Issue by cions** [denoland/rusty_v8#1248]
Cannot build with Rust nightly because TypeId switches to u128 in [this commit](https://github.com/rust-lang/rust/commit/9e5573a0d275c71dce59b715d981c6880d30703a).

```
    --> src/isolate.rs:1715:3
     |
1715 |   assert!(size_of::<TypeId>() == size_of::<u64>());
     |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'assertion failed: size_of::<TypeId>() == size_of::<u64>()', src/isolate.rs:1715:3
     |
     = note: this error originates in the macro `assert` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by bartlomieju** [comment]
Thanks for bringing this up. We do not target Rust nightly, but we'll make sure to upgrade this line once this change makes it to Rust stable channel.

**Comment by edoardopirovano** [comment]
@bartlomieju @mmastrac Any chance you'd be able to do a release to get this out? Our build is currently blocked on this and we'd love to avoid having to pin to a specific commit to get the fix.

**Comment by bartlomieju** [comment]
@edoardopirovano we are OOO this week, I can cut a release over the weekend.

**Comment by bartlomieju** [comment]
@edoardopirovano FYI I'm waiting for https://chromium.googlesource.com/v8/v8/+/ff4d0e4c53cbdf228c8245cdb959c0ffc3af5b62 to be available in `11.6-lkgr` branch to do the release as it will (hopefully) fix a painful bug from a few months back.
